cli/src/init/noir.rs

```diff
@@ -0,0 +1,17 @@
+use super::ProvingSystem;
+use anyhow::Result;
+pub struct Noir;
+
+impl ProvingSystem for Noir {
+    fn lib_template(_file_path: &str) -> Result<()> {
+        Ok(())
+    }
+
+    fn dep_template(_file_path: &str) -> Result<()> {
+        Ok(())
```

Comment:
we need this
```toml
noir = { git = "https://github.com/zkmopro/noir-rs", features = ["barretenberg", "android-compat"] }
```

Reply:
It works well on my side without it, and I'm assuming this dependency is embeded in `morpo-ffi`, isn't it?